src/main/java/cloud/fogbow/ras/core/plugins/interoperability/cloudstack/compute/v4_9/CloudStackComputePlugin.java

```diff
@@ -72,33 +85,20 @@ public boolean hasFailed(String cloudState) {
     }
 
     @Override
-    public String requestInstance(ComputeOrder computeOrder, CloudStackUser cloudUser) throws FogbowException {
-        String templateId = computeOrder.getImageId();
-        if (templateId == null || this.zoneId == null || this.defaultNetworkId == null) {
-            LOGGER.error(Messages.Error.UNABLE_TO_COMPLETE_REQUEST);
-            throw new InvalidParameterException();
-        }
+    public String requestInstance(@NotNull ComputeOrder computeOrder, final CloudStackUser cloudUser)
+            throws FogbowException {
 
+        checkCloudUser(cloudUser);
```

Comment:
Is this verification really necessary? I think _cloudUser_ should not get null to the plugin, and if it did, it would probably be by mistake in the configuration, which would not allow the instantiation of the plugin either.

Reply:
This would be my next question. If the Clouduser is not null, it is not necessary.